src/components/ec-inline-input-field/components/edit/edit.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div
+    class="tw-w-full tw-flex"
+    data-test="ec-inline-input-field-edit"
+  >
+    <ec-input-field
+      ref="input"
+      v-model="value"
+      type="text"
+      data-test="ec-inline-input-field-edit__input"
+      @keydown.enter="submitViaKeyboard"
+      @keydown.esc="cancelViaKeyboard"
+    />
+    <div class="tw-flex tw-ml-20 tw-mt-12">
+      <button
+        class="ec-inline-input-field-edit__action"
+        data-test="ec-inline-input-field-edit__submit-action"
+        @click="submit"
+        @keydown.enter.space.prevent="submitViaKeyboard"
```

Comment:
You don't need this function as well.  keep `@click="submit"` and you can check in submit that $event has a `code` prop set. keydown fires KeyboardEvent which has it, mouse click fires MouseEvent which doesn't.

Reply:
This functionality was removed (reason explained in a previous comment). Anyway, like curiosity, it seems that if you want to use the `$event.code` you need to specify the type of event within the tag. Otherwise (if we only define the `@click`, the callback is called, but even when it is done through the keyboard, what is received is finally a `MouseEvent` (instead of a `KeyboardEvent`).